exercises/03-is_prime.py

```diff
@@ -1,12 +1,13 @@
 def is_prime(n):
-    for i in range(2, n):
-        if n % i == 0:
-            return False
-    return True
-
+    if n > 1:
```

Comment:
Here you are making a special case for when `n <= 1`, and only run the algorithm when `n > 1`. This is fine, but another way to do this is to use a "guard clause", where you handle the special cases at the beginning of the method.
So, for example, you could do:
```
if n <= 1:
    return False

for i in range(...):
...
```
That way, you don't need to indent the main code. Some people don't like too many guard clauses, but in this case I think it's good.